spec/fixtures/test/manifests/top_scope_variables.pp

```diff
@@ -0,0 +1,4 @@
+# foo
```

Comment:
Can you remove this?

Reply:
I can remove it, but the test will fail with `[documentation] defined type not documented`.

Reply:
Also, I simply copied this `# foo`  header from other similar test manifest fixtures, eg. `spec/fixtures/test/manifests/fail.pp` and `spec/fixtures/test/manifests/init.pp` and others.

Reply:
We are trying to avoid this type of meaningless documentation. If a one-liner description doesn't make sense for this function then feel free to remove the entire comment and let Rubocop add it to the list of exceptions by running `bundle exec rubocop --auto-gen-config`.